pkgs/servers/http/nginx/generic.nix

```diff
@@ -166,6 +166,9 @@ stdenv.mkDerivation {
   preInstall = ''
     mkdir -p $doc
     cp -r ${nginx-doc}/* $doc
+
+    mkdir -p $man/share/man/man8
+    cp man/nginx.8 $man/share/man/man8
```

Comment:
This should use installManPage and probably be conditional on if the file exists

Reply:
```suggestion
    [[ -e man/nginx.8 ]] && installManPage man/nginx.8 
```

Reply:
Oh, great suggestion! I'll try to make it conditional with a TODO.